.github/workflows/ci.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
           sudo apt-get install python3-pip arping ndisc6
           sudo pip3 install invoke semver pyyaml
           go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0
+          go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.3
```

Comment:
Intentionally pinned it to a specific version.

Reply:
That's totally legit

Reply:
I think you need to add this here https://github.com/metallb/metallb/blob/7b746bb2bc940e4903aee3a1e65188f895247e82/.github/workflows/composite/setup/action.yaml#L20 rather than this place (here it's for unit tests)

Reply:
Added the fix.